Fabric.Authorization.API/scripts/Install-Authorization-Utilities.tests.ps1

```diff
@@ -385,6 +385,42 @@ Describe 'Get-SamAccountFromAccountName tests' -Tag 'Unit'{
     }
 }
 
+Describe 'Get-SamDomainFromAccountName tests' -Tag 'Unit'{
+    InModuleScope Install-Authorization-Utilities{
+        It 'Should parse a valid account domain'{
+            #Arrange
+            $accountName = "domain\test.user"
+
+            #Act
+            $samAccountName = Get-SamDomainFromAccountName -accountName $accountName
+
+            #Assert
+            $samAccountName | Should -Be "domain"
+        }
+        It 'Should throw an exception for an invalid account domain'{
+            #Arrange
+            $accountName = "domain\test\user"
+
+            #Act
+            {Get-SamDomainFromAccountName -accountName $accountName } |Should -Throw
+        }
+        It 'Should throw an exception for a null account domain'{
+            #Arrange
+            $accountName = $null
+
+            #Act
+            {Get-SamDomainFromAccountName -accountName $accountName } |Should -Throw
+        }
+        It 'Should throw an exception for a empty account name'{
```

Comment:
One more potential test case - account name without the domain should also throw.

Reply:
Added that